lib/reply.js

```diff
@@ -76,6 +84,11 @@ Reply.prototype.code = function (code) {
   return this
 }
 
+Reply.prototype.serializer = function (fn) {
+  this._serializer = fn
+  return this
+}
```

Comment:
I'm rewriting the docs, in the next week will land in pr.
If you are ok we can merge this as it is, and go deeper with this argument in the new docs.